src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -34,13 +34,15 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var childTags = descriptor[nameof(TagHelperDescriptor.AllowedChildTags)].Value<JArray>();
             var documentation = descriptor[nameof(TagHelperDescriptor.Documentation)].Value<string>();
             var tagOutputHint = descriptor[nameof(TagHelperDescriptor.TagOutputHint)].Value<string>();
+            var caseSensitive = descriptor[nameof(TagHelperDescriptor.CaseSensitive)].Value<bool>();
             var diagnostics = descriptor[nameof(TagHelperDescriptor.Diagnostics)].Value<JArray>();
             var metadata = descriptor[nameof(TagHelperDescriptor.Metadata)].Value<JObject>();
 
             var builder = TagHelperDescriptorBuilder.Create(descriptorKind, typeName, assemblyName);
 
             builder.Documentation = documentation;
             builder.TagOutputHint = tagOutputHint;
+            builder.CaseSensitive = caseSensitive;
```

Comment:
It's nice that we use the builder here, or else we'd have to update all of these serializers 😆